raiden/utils/cli.py

```diff
@@ -201,6 +202,27 @@ def convert(self, value, param, ctx):
             self.fail('Please specify a valid hex-encoded address.')
 
 
+class LogLevelConfigType(click.ParamType):
+    name = 'log-config'
+    _validate_re = re.compile(
+        r'^(?:'
+        r'(?P<logger_name>[a-zA-Z0-9._]+)?'
+        r':'
+        r'(?P<logger_level>(?:debug|info|warn(?:ing)?|error|critical|fatal))'
+        r',?)+$',
+        re.IGNORECASE
+    )
+
+    def convert(self, value, param, ctx):
+        if not self._validate_re.match(value):
+            self.fail('Invalid log config format')
+        level_config = dict()
+        for logger_config in value.split(','):
+            logger_name, logger_level = logger_config.split(':')
```

Comment:
what if the user provided an invalid string? e.g. missing the colon `raiden.raiden_service`, will convert be called or is the `_validate_re` expected to catch it?

Reply:
It's validated right [above](https://github.com/raiden-network/raiden/pull/1464/files#diff-8b6c0296d86609e6cad423a1e829db68R218). So `validate_re` will catch it.